site/content/architecture/component-view/top-level.xml

```diff
@@ -0,0 +1 @@
ame="Page-1">7Z1bc5s4FIB/jR83gySuj4mTtjvT7KSbzGz3kRjVZouRF+Nc9tevhCUMkpwYm0tjlIfWyCDgfEfnopsnaLp8+ZyFq8UtiXAygVb0MkHXEwiBH9j0P1byui2BwLO2JfMsjvhZu4L7+D/MC8VpmzjC69qJOSFJHq/qhTOSpniW18rCLCPP9dN+kKR+11U4x0rB/SxM1NK/4ihf8FLXsXdffMHxfCFuDdxg+81jOPs5z8gm5TecQPSj+Nt+vQxFZfxN14swIs+VInQzQdOMkHz7afkyxQmTrpDb9rpPe74tHzzDaX7QBQhtL3kKkw1/+6vNOk7xej1h17r0DSgWdHW5WiXxLMxjkvJHz1+FvOhbrNjHGVmuSFrc+ipM4nlKyxL8gx2uV+EsTudfi6Nr5NKiQkyYPQigR/MkXDNs7F7rPCM/8ZQkJCtugHz4iFx2zY84SSrlkYP9yGblJM25HkGHHvOXwlmOX/aKBpQCp6qMyRLn2Ss9RVwAEYfE1dgW6vm80wno8rJFRR3KwpDr4bysfIeCfuA09pGxFTJ7BZ/hfzdxxprMVUT/nxWU0HVKMvagV4t8mXAxNxC6VfypQucKrQp9W4NoMaz+Fc5i+uY4u9/i58Vt0AGOV6djA4VOoIGD/DbYuIZNAzYO7JONZ9g0YONqrFp3bILD2SQkSeIVWUls1mTTBhtV5ofSagOCFUgNxPIuHAWDzre4TgsYbMtg0GII3F4xAAXD5e23O1oCLqqh1y0NxsI5MwPQ+gPnzyT7Wfl2HNGYD6RgzIJqMOZ7XQVjNlRQfXl4uJtM4eSyFiVH4Yqa7UHxNPY/bTQlkQC9CjejehWga0gAtoLH+fgGLSN5yJ/F6si+2bBXN+OoVK7xUzzDSrP5E8/jdVH9yNoNQvV207NZ89RA4JI2A/rwZapfArnH2dMYLZvkd1w1lYFWZ4bNU0OEvWL/YMlMxd79BvyWTJ6My7GQgkvXnFrJbjw1SPjIXqhFKgGsNyI/UL0Q6sYLQctVIOBoju/5IWsdZE7SMLnZlVaEbtVp4Jc4/175/Dc75YJrfpjll6z7uWh1KRZln2L2dBwZPebyBgGrI42kS2hJ5QJ6JE5H9PAfnOevvCDc5IQW7V7gK2H6VDyaRgG0ppFJoinUDCdURZ7q1+kQ8UvvSExrLJXB8etN1LetCx/t/iTmVGBznPM6YKUnW6oWweACuLs/z25yF9rushlW7lIoV/nqh+kbcLrXN88pVIM+1Pfi2LF9UbA9IbCRKLgT5reqgkZN31PTd/QpgMfok3IXYDmgSb17WsMRego8dXzh9tvDg0lp96S0Puw1pRURybkEE0FLsYQvjcqVGVQvGa2lMDjBthtLfJAlpjayjrxMWxvHCOiditoLA0rr0XkYYLToYH8OrcqfU1MFKDpemnpwxwdvV7RHOan4w9fKaSt2wvqtsFl/n52Kbms8WmFtdSaIcTjM4fgDOhy7wSSQMUEBFhySyvlO/+gGlmxsfbWXu7M5B/b5zgfpxNy5Q7JqkPQYVtAHQ7I6YC6PGmbW0p/3gkUR9VoX4kj0brn8WO7cmhwYXtKnLOK7mt6pcq7I0dHIUZSdGJr6SMKIDktL1Irc4xKlY8JFB35g/EORDqThKegEbfQiytV2yv2ANMFwf68LoyXwR3eNHEW+QdQ7plwEufXMvN8eyQbR7Zig2NAZEIrpu9dCcZwhW0qDuecmt4COEqqouUU5g7kKy7ZagCW0wMA6zAX5SIKlTvMs+xJbzwSF/zOwDoMFguFglXMVT/BNLcH6tXyTHMW5Tn++CVoNWtCYoNgQDQjlgB6OMUKRo7h+oTQYpdzja1oKrT+Er3GkYTJPgKn6GthRFAet05ewjwkWkqZJ62Fp5km3Exg0WENlYClziXqGdXq3z1n6Jh8N6ZtO7/Y5TyhecOEMiOX0TQfOEguw4ICNBZzewzMmdxNIAz2+pbqbMo1sPZADp/fwjAmWb/mSwdPj6io6AA0yV4NrF7oNA0sdVi22lih2X0vY2pnHjH6as098+fw+lptlcjnLmbCvmEDiWZh8DR9xckfWMRfaI8lzsqQnJOyLq3K7Oh0kUcclX8qTF37xI+/4Ykuj5ZolPbbG4QGrFSN6BpsfdbI8GAhBCCxIu01FVwuE4TjGmlrD5dY3FQGagUFd1NiKsYRqIFIsWoRuuGSNJX1cr4q3NJazU8tZrsLrxXKKBmks5zuW07f7tZynd8aP2XL6bp+WU+2c2lpOiRd9u3yimbAppMsndFYFzovEiu8ZlQauWlJhBZdxFBXLHJ8XcY6ZvNk9n7NwVaNuSdQrDIu1i220G0siYaskdOasjZXd5ezutzZdkrzXOClBr+50HM0Wv91RUiONbVpm2OhaENBuLdcdHTUimGY4Ym0oTNYGEnccoutbzLeEagdvd4jUwd8HnIZsHrqhU0je8waks3dvRoUO35zRuKZqJCcF3v2yU3sMjfHTTLOsI/KA3SMidXzeGL+68fMHpKPmQsb4NTF+zoDs1F70KVkuwzTS4eHdgtt/6VVTKo2MJIYkJydtKtOrG9P8CoGxkbUxf2ldeL901Pz3ASeYV2QAbZNg3x6QkCYHNpawJUvYq0/TbL1lLOFblrBfOrqODGMJ37aE/RJSOzOMJTwaJZDGem1o9dm1q9lc7eYJG1u4l4/uNw66o6Pr2DDG8B1CYhOuXggdsAqhPsSuE1iF0Jsj8JEXPB46MwK7s1lHEofy/jqqwJF2fkobP6MDD9kKrZ95EEfNTykoVCY8tDc9BUmbSHgaLF21A0ftXtgL5RecNNRa05AZ2Kov1zEArTBQU9S9DM7q5woBlHYN1k011m3N2MpyDc0ebXcZeYqZwKB1eff7SP2yrazLsFUq5c/WtW+RNON2JF1vlgaLhEXjJzrEooa0Ny9UaGnITnvrd+/HgQf5rmTKfIWO9ifwjqBDDzNC8sp3n+krL25JhNkZ/wM=</diagram></mxfile>
```

Comment:
It is actually the Device Registry that uses the Auth Server instead of the other way around ...

Reply:
Indeed. fixed it